content/release-10-17-0/devices-10-17-0.md

```diff
@@ -204,6 +204,13 @@ mandatory <b>Software type</b> field is filled.</td>
 <td>10.16.260.0</td>
 </tr>
 
+<tr>
+<td>Device Management</td>
```

Comment:
@dmajsag The improvements and fixes are taken from the Jira tickets, so no need to add them here via PR.